webserver/shared/constants.ts

```diff
@@ -29,5 +29,10 @@ export const DEX_PRICE_LIMIT = {
 
 export const POOL_DELEGATION_LIMIT = {
   POOLS: 50,
-  SLOT_RANGE: 200,
+  SLOT_RANGE: 20000,
+};
+
+export const PROJECTED_NFT_LIMIT = {
+  SLOT_RANGE: 100000,
```

Comment:
I feel like paginating by slot range like this is a really bad idea. It means the time it takes to query projected NFTs grows as the blockchain grow instead of growing as the number of projected NFTs grow leading to a bunch of useless queries where you query an slot range that is empty